interop-testing/src/main/java/io/grpc/testing/integration/TestServiceImpl.java

```diff
@@ -447,6 +447,7 @@ private StreamingOutputCallResponse toResponse() {
   /**
    * Creates a buffer with data read from a file.
    */
+  @SuppressWarnings("Finally") // Not concerned about suppression; expected to be exceedingly rare
   private ByteString createBufferFromFile(String fileClassPath) {
```

Comment:
Currently, no, we can't do java7 in test code. This is actually something that became harder with the new animalsniffer plugin, but the old animalsniffer plugin was worrying me because it missed some things.